src/Illuminate/Auth/Console/AuthMakeCommand.php

```diff
@@ -63,6 +64,28 @@ public function handle()
             );
         }
 
+        if (! $this->option('no-tests')) {
```

Comment:
`\Illuminate\Auth\Console\AuthMakeCommand::handle` is really big now.
Maybe extract that to a `makeTests` method and call if if `! $this->option('no-tests')` ?